camunda-formio-tasklist-vue/docs/components/FormViewSubmission.md

```diff
@@ -0,0 +1,26 @@
+# FormViewSubmission Component
+
+Customised component for viewing FormSubmissions in Application
+History.
+
+## Props
+
+| Name | Type |  Default | Description |
+| --- | --   | --- | ------- |
+|  formId  :triangular_flag_on_post: | string | undefined | Specify the formid of formio form submission you want to view.|
+|  submissionId  :triangular_flag_on_post: | string | undefined | Specify the formid of formio form submission you want to view.|
+
+## Usage
+
+```
+<template>
+    <FormViewSubmission
+      v-if="formId && submissionId"
```

Comment:
no need of  `v-if="formId && submissionId"`

Reply:
instead of  passing dynamic variable, for these examples, it will be better to give static sample values

Reply:
```
import FormViewSubmission from "camunda-formio-tasklist-vue/src/components/FormViewSubmission.vue";

<FormViewSubmission
  :formid="abc-f563-gt66-g6754"
  :submissionid="abc-hy64f-g78fcf-d35fd"
/>
```
Is Usage as this way fine?